src/lib/__tests__/server-utilities.test.js

```diff
@@ -30,3 +35,21 @@ describe('md5Hash', () => {
     expect(fileNameHash).not.toBe(md5Hash(fileName));
   });
 });
+
+describe('compressJPEG', () => {
+  if (!fs.existsSync('./tmp')) fs.mkdirSync('./tmp');
+  fs.closeSync(fs.openSync('./tmp/test.jpg', 'a')); // Create empty file 'test.jpg'
+  const filePath = './tmp/test.jpg';
+
+  it('overwrites uncompressed image with compressed image', () => {
+    compressJPEG(filePath).then(() => {
+      expect(fs.existsSync(filePath)).toBe(true);
+    });
+  });
+
+  it('deletes image', () => {
```

Comment:
I'm pretty sure (though we're currently using an old version of Jest that for example needs the annoying .unmock statements, I'm gonna upgrade to the newest Jest in the very near future before our test suite becomes too big where this comment is definitely correct) that you need to take the `done` argument from `it` here and call it when it's done when doing asynchronous tests. Also the same above, though it seems to be currently working... I'm not sure if that's simply because it's such cheap operations since you're working on an empty file

Reply:
Okay I just checked [the old docs](https://github.com/facebook/jest/blob/v13.2.3/docs/TutorialAsync.md), you just need to make sure you return a promise, so add a return statement both here and above.

Reply:
I'm not really sure what you mean here. Where do you think I should put a return statement?

Reply:
Did you check the link I named 'the old docs' above? That's the tutorial for async tests in Jest v13